lib/src/main/java/com/springboot/webpush/common/api/Action.java

```diff
@@ -1,43 +1,79 @@
 package com.springboot.webpush.common.api;
 
+import java.util.List;
+
+/**
+ * The action class, specific properties to be used to send the {@link PushMessage} to the service.
+ */
 public class Action {
```

Comment:
I think this class corresponds to this spec: https://notifications.spec.whatwg.org/#dictdef-notificationoptions

Might be good to keep names the same as much as possible

Reply:
cleaned up on other PR